custom_components/renault/sensor.py

```diff
@@ -69,7 +69,8 @@ async def get_vehicle_entities(hass, vehicle_proxy: PyzeVehicleProxy):
         RenaultOutsideTemperatureSensor(vehicle_proxy, "Outside Temperature")
     )
     entities.append(RenaultPlugStateSensor(vehicle_proxy, "Plug State"))
-    entities.append(RenaultRangeSensor(vehicle_proxy, "Range"))
+    entities.append(RenaultRangeBatterySensor(vehicle_proxy, "Battery Range"))
```

Comment:
Can you make use of MODEL_USES_FUEL here to create either the battery OR the fuel sensor?